apps/dbmgr.py

```diff
@@ -103,10 +103,20 @@ def frames(self):
         """Overridden."""
         return (self.managerFrame,)
 
-    def sendDB(self):
-        """Emits a broadcastRequested signal with the database list."""
+    def sendDB(self, isAdded: bool, name: str):
+        """Emits a broadcastRequested signal with the database list and a logging message.
+        
+        Args:
+            isAdded: True if a database is added, otherwise False.
```

Comment:
It depends on context, if the "otherwise" means a database is removed.
In this case, I think you should explicitly write that `False` is for the case when a database is removed.

Reply:
> > But, I know if you used the closing keyword in commit messages, the issue will be closed automatically. Isn't it enough?
> 
> Oh, I wanted to talk about that. As you have experienced before, commit messages do not strongly belong to this PR. For example, if you do rebase or cherry-pick, commits can be transferred to other PRs. In this way, issues can be closed unexpectedly when the other PR is merged.
> 
> Therefore, I would like to suggest that we _never_ write the closing keywords in commit messages, but in the PR main article. It also explicitly shows which issues will be closed. Note that people barely read the commit messages. Moreover, it does not appear on "Development" section of this PR (at the right of the page).

I didn't realize that problem. Then, we will never write the closing keywords.